addon/components/paper-dialog.js

```diff
@@ -42,20 +41,21 @@ export default Component.extend({
       return '#ember-testing';
     }
     let parent = this.get('defaultedParent');
-    let $parent = $(parent);
-    // If the parent isn't found, assume that it is an id, but that the DOM doesn't
+    let parentEle = document.querySelector(parent);
```

Comment:
Without jQuery the proper  way is to test if 'parent' is a string or an element.

You could do:

```js
if (typeOf(parent) === 'string') {
  // according to the docs, it's either an element or a selector, no need to test for '#'...
  return parent;
} else {
  //
} 

```

Reply:
What about the jQuery objects? can I just assume strings or element and update docs?